invenio_app_ils/ui/src/invenio_app_ils/common/helpers.js

```diff
@@ -0,0 +1,7 @@
+export const shortenString = input => {
```

Comment:
Another option is to use lodash [truncate](https://lodash.com/docs/4.17.11#truncate).

Reply:
Thanks, I didn't know about it!